.circleci/values/hydra.yaml

```diff
@@ -18,6 +18,14 @@ hydra:
         - "foo bar 123 456 lorem 3"
 maester:
   enabled: false
+
+deployment:
```

Comment:
Sorry for the delay on this PR (I only do this on weekends). I've added this file change too!